### PR TITLE
Further documentation to get a user started

### PR DESCRIPTION
**Pull request by krishnadurai**
This PR adds further documentation for users to understand aiXplain and aiXplain pipelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,32 +3,37 @@
 aiXplain Pipelines enables python programmers to add AI functions
 to their software.
 
+An aiXplain pipeline is a directed acyclic graph (DAG) of AI functions built using aiXplain's designer UI. An AI function is a data processing step that relies on a machine learning model to execute. An example of an AI function is speech recognition or machine translation. It helps you process your data by calling a series of functions as defined in the DAG, abstracting the orchestration by providing a simple python function call.
+
+aiXplain has a collection of AI models for each AI function. You can explore the collection of our AI models by using the discover feature of our [platform's website](https://platform.aixplain.com/).
+
+## aiXplain Pipeline Designer DAG
+
+The image below shows a sample aiXplain pipeline built for subtitling video files.
```
Comment (thiago-aixplain) [inline README.md:12]: Maybe add a short description of the pipeline:

The pipeline received an English video as input and first extracts its audio, which is then fed to a VAD node, responsible to find the voice activity segments of the audio input. These segments are then transcribed by a Speech Recognition node and then translated into Spanish by a Translation English-Spanish node. Finally, a Speech Synthesis node is added to dub the Spanish translation.

File: README.md [hunk 2]
```diff
@@ -3,32 +3,37 @@
 aiXplain Pipelines enables python programmers to add AI functions
 to their software.
 
+An aiXplain pipeline is a directed acyclic graph (DAG) of AI functions built using aiXplain's designer UI. An AI function is a data processing step that relies on a machine learning model to execute. An example of an AI function is speech recognition or machine translation. It helps you process your data by calling a series of functions as defined in the DAG, abstracting the orchestration by providing a simple python function call.
```
Comment (thiago-aixplain) [inline README.md:6]: The text is clear to technical readers. Perhaps Dan can help us in simplifying the text to be clear also for non-technical readers.

File: README.md [hunk 3]
```diff
@@ -3,32 +3,37 @@
...
+
+<img src="docs/assets/designer-subtitling-sample.png" width=30% height=30%>
+
+
 ## Installation
 
 ```
 pip install aixplain-pipelines
 ```
-## Developer Instructions
```
Comment (thiago-aixplain) [inline README.md:11]: We may need a section showing a code snippet to call a pipeline:

```python
from aixplain_pipelines import Pipeline

pipelines_url="ADD ENDPOINT URL HERE"
api_key="ADD API KEY HERE"

pipe = Pipeline(api_key=api_key, url=pipelines_url)

path="ADD LINK TO THE INPUT DATA HERE"
response = pipe.run(data=path)
```

File: README.md [hunk 4]
```diff
@@ -3,32 +3,37 @@
...
 
-### Installation
```
Comment (thiago-aixplain) [inline README.md:13]: We may also want to show an example of the output:

```
{'success': True,
 'response': {'status': 'SUCCESS',
  'data': 'https://modelserving-data.s3.amazonaws.com/0279dc6e-8e55-45a4-88b1-dc7242e908a1.out',
  'completed': True,
  'numofsegments': 4,
  'numoffailedsegments': 0,
  'progress': 100,
  'segments': [{'index': 0,
    'separator': ' ',
    'size': 1499,
    'inputs': {'1': {'value': 'https://aixplain-vad.s3.amazonaws.com/6026307d-70c2-4070-9f09-a1ad1d4dc540_0.txt',
      'input_type': 'text'}},
    'success': True,
    'response': 'https://modelserving-data.s3.amazonaws.com/868bc682-b0e8-478c-9158-0f60a43e64a3.tmp',
    'output_type': 'text',
    'output_name': '3_ASSET',
    'details': '',
    'credits_used': 0.01499,
    'timeSpent': 0.969},
   {'index': 1,
    'separator': '\n',
    'size': 101,
    'inputs': {'1': {'value': 'https://aixplain-vad.s3.amazonaws.com/6026307d-70c2-4070-9f09-a1ad1d4dc540_1.txt',
      'input_type': 'text'}},
    'success': True,
    'response': 'https://modelserving-data.s3.amazonaws.com/ca20396d-fb04-41eb-9b48-b253e1a0692d.tmp',
    'output_type': 'text',
    'output_name': '3_ASSET',
    'details': '',
    'credits_used': 0.00101,
    'timeSpent': 0.925},
   {'index': 2,
    'separator': ' ',
    'size': 1494,
    'inputs': {'1': {'value': 'https://aixplain-vad.s3.amazonaws.com/6026307d-70c2-4070-9f09-a1ad1d4dc540_2.txt',
      'input_type': 'text'}},
    'success': True,
    'response': 'https://modelserving-data.s3.amazonaws.com/941e6a08-f814-4ef7-85a0-fbfb5abcc4e6.tmp',
    'output_type': 'text',
    'output_name': '3_ASSET',
    'details': '',
    'credits_used': 0.01494,
    'timeSpent': 0.938},
   {'index': 3,
    'separator': '\n',
    'size': 56,
    'inputs': {'1': {'value': 'https://aixplain-vad.s3.amazonaws.com/6026307d-70c2-4070-9f09-a1ad1d4dc540_3.txt',
      'input_type': 'text'}},
    'success': True,
    'response': 'https://modelserving-data.s3.amazonaws.com/b26a9055-0fa0-4612-be3b-c62511b616ab.tmp',
    'output_type': 'text',
    'output_name': '3_ASSET',
    'details': '',
    'credits_used': 0.00056,
    'timeSpent': 0.782}]},
 'error': None,
 'elapsed_time': 0.609717845916748}
```

File: README.md [hunk 5]
```diff
@@ -3,32 +3,37 @@
...
 
-### Run pytest
+This demo creates a `.srt` file for the supplied video using aixplain-pipelines. Follow the instructions in the [documentation](docs/samples/subtitle_generator/README.md).
 
-```
-pytest
-```
+## Developer Guide
+
+Follow the developer guide [documentation](docs/developer_guide.md).
```
Comment (thiago-aixplain) [inline README.md:36]: The link is off, right?

Comment (krishnadurai) [reply README.md:36]: Good catch Thiago!

**Comment by krishnadurai** [comment]
@daninelson please do a post review after my merge.

File: README.md (hunk 2 above)
Comment (daninelson) [reply README.md:6]: The text reads fine to me. But maybe "abstracting the orchestration" may not translate well for non-English speakers, maybe changing this to something different? (not sure what)

**Pull request merged**